src/api_server/Cargo.toml

```diff
@@ -23,4 +23,4 @@ utils = { path = "../utils" }
 vmm = { path = "../vmm" }
 
 [dev-dependencies]
-libc = "0.2.117"
+libc = "0.2.117"
```

Comment:
nit: new line at the end of the file

Reply:
I will check the other tomls to confirm, I can re-add it back in [#3792](https://github.com/firecracker-microvm/firecracker/pull/3792).